raiden/utils/__init__.py

```diff
@@ -36,6 +36,22 @@ def random_secret():
     return os.urandom(32)
 
 
+def decode_hex(s) -> bytes:
+    if isinstance(s, str):
+        return bytes.fromhex(s)
+    if isinstance(s, (bytes, bytearray)):
+        return unhexlify(s)
+    raise TypeError('Value must be an instance of str or bytes')
+
+
+def encode_hex(b) -> str:
+    if isinstance(b, str):
+        b = bytes(b, 'utf-8')
+    if isinstance(b, (bytes, bytearray)):
+        return str(hexlify(b), 'utf-8')
+    raise TypeError('Value must be an instance of str or bytes')
```

Comment:
Looks like there's some kind of intersection between these functions and `(address|data)_(en|de)coder`. If so, wouldn't be useful to keep/merge the most broad/complete implementation?

Reply:
Yeah, I tried to keep the diff as small as possible for now. In the long term we should replace this with functions from `eth_utils`. I'll create issues for this.